File: jft.academia.v01/Engine/jft.academia.v01.Aplicativo/ModelsEntities/Atividades_GruposModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace jft.academia.v01.Aplicativo.ModelsEntities
{
    public class Atividades_GruposModel
    {

        public int id_atividade_grupo { get; set; }
        public int id_atividade { get; set; }
        public int id_grupo_atividade { get; set; }        
        public int nr_ordem { get; set; }
        public string te_descricao { get; set; }


    }
}

File: jft.academia.v01/Engine/jft.academia.v01.Core.EntitiesViews/Atividades_GruposModel.vb
Public Class Atividades_GruposModel
    Public Sub New()
        'Atividades = New AtividadesModel()
        'Grupos = New GruposModel()
    End Sub


    Public Property id_atividade_grupo As Integer
    'public int id_atividade { get; set; }

    Public Property Atividades As AtividadesModel

    'public int id_grupo_atividade { get; set; }

    Public Property Grupos As GruposModel


    Public Property nr_ordem As Integer
    Public Property te_descricao As String

End Class
